app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsFragment.kt

```diff
@@ -33,9 +33,18 @@ class ChatRoomsFragment : Fragment(), ChatRoomsView {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         presenter.chatRooms()
+        floating_search_view.setOnQueryChangeListener { oldQuery, newQuery ->
```

Comment:
Shouldn't the `floating_search_view` name be camelCase? 🤔

Reply:
Usually yes. But in this case it's in snake_case because of the Kotlin plugin we use: https://antonioleiva.com/kotlin-android-extensions. What we should agree is to name our view ids in camelCase since we're using this extension.

Reply:
Stepping in for @leonardoaramaki  the convention for our xml files is right, but because we use Kotlin Android Extension we don't need to use the known findViewById, get the result and assign to a variable (like `floatingSearchView`). We only need to refer the xlm id (`floating_search_view`) and Kotlin does the rest (find it, cache it...)